clean_code_main/clean_code_checks/y_check_equals_sign_chaining.clas.abap

```diff
@@ -28,7 +28,7 @@ CLASS Y_CHECK_EQUALS_SIGN_CHAINING IMPLEMENTATION.
     settings-pseudo_comment = '"#EC EQUALS_CHAINING'.
     settings-documentation = |{ c_docs_path-checks }equals-sign-chaining.md|.
     settings-disable_threshold_selection = abap_true.
-    settings-threshold = 1.
+    settings-threshold = 0.
```

Comment:
Is there any impact on customers who are using this check already?

Reply:
looks ok